tests/test_parsing/test_reqs.py

```diff
@@ -0,0 +1,139 @@
+"""Check the requirements parsing."""
+import io
+
+from pkg_resources import parse_requirements
+from fades import parsing, REPO_PYPI, REPO_VCS
+
+
+def get_req(text):
```

Comment:
Don't use this get_req, but the central one I normalized in the previous branch (that would lead you to remove the "list around the get_reqs call in the rest of the file).

Thanks!